util/warnings_to_filter.txt

```diff
@@ -137,7 +137,6 @@ warning: return type of member DYN::ModelCPP::initSubBuffers is not documented
 warning: return type of member DYN::ModelCPP::loadParameters is not documented
 warning: return type of member DYN::ModelCPP::loadVariables is not documented
 warning: return type of member DYN::ModelCPP::notifyTimeStep is not documented
-warning: return type of member DYN::ModelCPP::printInitValues is not documented
```

Comment:
How come this one is deleted and not modified?

Reply:
The DYN::ModelCPP::printInitValues method doesn't exist.

Reply:
ok, what about the one in SubModel?

Reply:
It seems the doxygen bug occurs with virtual methods only. SubModel::printModelValues doesn't trigger any warning with doxygen.